db/schema.rb

```diff
@@ -33,6 +33,18 @@
     t.index ["calendar_id"], name: "index_events_on_calendar_id"
   end
 
+  create_table "installs", force: :cascade do |t|
```

Comment:
What is this? And where do you use this model?

Reply:
I haven't added this to be honest